redcap_bridge/utils.py

```diff
@@ -122,3 +122,33 @@ def remove_columns(csv_file, compressed_file=None):
 def exportCSVtoXLS(csv_file, compressed_file=None):
     read_file = pd.read_csv(csv_file, na_filter=False, dtype='str')
     read_file.to_excel(r'Path', index=None, header=True)
+
+def conversion_csv_to_json(csv_file):
+    """
+    Test conversion function
+    """
+    df = pd.read_csv(csv_file)
+    df.to_json(r'/Users/killianrochet/PycharmProjects/DigLabTools/redcap_bridge/test_redcap/testfiles/elabConversion/pandaJsonConversion..json')
+    json_df = pd.read_json('/Users/killianrochet/PycharmProjects/DigLabTools/redcap_bridge/test_redcap/testfiles/elabConversion/pandaJsonConversion..json')
+    for i in json_df.columns:
+        for j in json_df.index:
+            match json_df[i][j]:
```

Comment:
Ohlala, you are using the latest Python features here (>=3.10). If we keep the match-case mechanism then we should also update our package requirements

Reply:
Ok i will change for a dictionary or just simple if statement